pkg/scheduler/scheduler.go

```diff
@@ -204,18 +204,12 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		ctx := ctrl.LoggerInto(ctx, log)
 		if e.assignment.RepresentativeMode() != flavorassigner.Fit {
 			if len(e.preemptionTargets) != 0 {
-				preempted, err := s.preemptor.IssuePreemptions(ctx, e.preemptionTargets, cq)
-				if err != nil {
-					log.Error(err, "Failed to preempt workloads")
-				}
-				if preempted != 0 {
-					e.inadmissibleMsg += fmt.Sprintf(". Pending the preemption of %d workload(s)", preempted)
-					e.requeueReason = queue.RequeueReasonPendingPreemption
-				}
+				e.inadmissibleMsg += fmt.Sprintf(". May require the preemption of %d workload(s)", len(e.preemptionTargets))
+				e.requeueReason = queue.RequeueReasonPendingPreemption
 			} else {
 				log.V(2).Info("Workload requires preemption, but there are no candidate workloads allowed for preemption", "preemptionReclaimWithinCohort", cq.Preemption.ReclaimWithinCohort, "preemptionWithinClusterQueue", cq.Preemption.WithinClusterQueue)
+				continue
 			}
-			continue
```

Comment:
Why do we need to admit workloads if there are preemption targets?
This means why do we need to move `continue` to L211?

Reply:
We don't fully admit them, we reserve the quota only. Previously we wold have keep the workload at the head of the queue until it wold have fit with no extra eviction needed.